bson/__init__.py

```diff
@@ -1336,3 +1337,24 @@ def decode(self, codec_options: "CodecOptions[_DocumentType]" = DEFAULT_CODEC_OP
 def has_c() -> bool:
     """Is the C extension installed?"""
     return _USE_C
+
+
+def _before_fork():
+    """
+    Acquires the ObjectId lock.
+    """
+
+
+def _after_fork():
+    """
+    Releases the ObjectID lock in parent and child.
+    """
```

Comment:
We can't release the lock in the parent, only the child.

Also can we fix the docstring format to be a single line?